### ISSUE TITLE
Spurious release tag release-4.7.0?

### ISSUE THREAD

**Issue by mjg** [darktable-org/darktable#15950]
### Describe the bug

There is a tag `release-4.7.0` for commit 40712e5ea3 which is contained in `release-4.6.0`. This looks unintentional and throws off automatic release monitoring by tags.

### Steps to reproduce

```
git tag --contains release-4.7.0
git log --decorate --boundary  release-4.7.0...release-4.6.0
````

### Expected behavior

I expect 4.7.0 to contain 4.6.0 (and not to be present yet).

### Commit

40712e5ea3

### Where did you obtain darktable from?

GitHub nightly

### darktable version

4.6.0

### What OS are you using?

Linux

### What is the version of your OS?

Fedora 39

### Describe your system?

n/a (issue is about tags on github)

### Are you using OpenCL GPU in darktable?

None

### Please provide additional context if applicable. You can attach files too, but might need to rename to .txt or .zip

I might be misunderstanding the branch/tag model of the project ...

**Comment by victoryforce** [comment]
> I might be misunderstanding the branch/tag model of the project ...

Darktable releases always have only an even minor version. The odd minor version is not intended for release, this is the base version of the development code. Development git snapshots get a version that consists of the base version plus the number of commits after the corresponding tag. For example, 4.7.0+59 means a build on commit 59 after setting the 4.7.0 tag.

> This looks unintentional and throws off automatic release monitoring by tags.

Automatic release monitoring should not consider odd minor versions.

**Comment by mjg** [comment]
Thanks for the clarification. `release-4.7.0` is a funny tag name for something that's not even an alpha release but a fork point, but now I know.

**Comment by victoryforce** [comment]
> `release-4.7.0` is a funny tag name

I have always thought that what is funny and what is not is very subjective. :)

> for something that's not even an alpha release but a fork point, but now I know.

Maybe you didn't understand something, or you formulated it poorly, because `release-4.7.0` is not a fork point, and not even a branch point. This is just a tag, that is, a pointer to a specific commit. It is used exclusively to generate versions of development snapshots in the style I described in the previous comment.

In other words, `<something>.<even>.0` is the final commit, it points to the release commit. `<something>.<odd>.0` is the initial commit that indicates the beginning (and is the basis for the generation of the version) of the next development cycle.
